RACK-Ontology/ontology/SYSTEM.sadl

```diff
@@ -10,67 +10,68 @@
  * of the Defense Advanced Research Projects Agency (DARPA).
  */
 
+/************** edit history *****************
+  *
+  * dwa - 11 Nov 2020 - removed incorrect usages of relationships
+  * dwa - 11 Nov 2020 - added that developedBy is a type of wasAssociatedWith
+  *
+  *********************************************/
+
 uri "http://arcos.rack/SYSTEM" alias sys.
 import "http://arcos.rack/PROV-S".
 import "http://arcos.rack/DOCUMENT".
 
-// Do we want to add MTBF, type (software, hybrid etc.,) to SYSTEM? They would be needed in an assurance case.
-
 SYSTEM
 	(note "SYSTEMs classify logical parts of the whole system being developed")
 	is a type of ENTITY.
 
-	partOf (note "Used to define a relationship to a higher level system") describes SYSTEM with values of type ENTITY.
-	partOf is a type of wasDerivedFrom.
+	partOf (note "Used to define a relationship to a higher level system") describes SYSTEM with values of type SYSTEM.
 
-	producedBy (note "ACTIVITY(s) participating in developing this SYSTEM") describes SYSTEM with values of type ACTIVITY.
+	producedBy (note "ACTIVITY(s) participating in developing this SYSTEM") describes SYSTEM with values of type SYSTEM_DEVELOPMENT.
+	producedBy describes SYSTEM with at most 1 value.
 	producedBy is a type of wasGeneratedBy.
 
-	provides (note "List of entities that this SYSTEM provides to others") describes SYSTEM with values of type ENTITY.
-	provides is a type of wasDerivedFrom.
+	provides (note "List of entities that this SYSTEM provides to others") describes SYSTEM with values of type FUNCTION.
 
-	requires (note "List of entities that this SYSTEM requires from others") describes SYSTEM with values of type ENTITY.
-	requires is a type of wasDerivedFrom.
+	requires (note "List of entities that this SYSTEM requires from others") describes SYSTEM with values of type FUNCTION.
 
-	function (note "functionality / capability of a SYSTEM") describes SYSTEM with values of type FUNCTION. 
-	function is a type of wasDerivedFrom. 
+	functionalityOverview (note "functionality / capability of a SYSTEM") describes SYSTEM with values of type string. 
+	functionalityOverview describes SYSTEM with at most 1 value.
```

Comment:
We could probably remove the cardinality constraint here without damaging too much, because it's unlikely that the content of functionalityOverview will be used in filtering data during a query.

Reply:
Let's close and delete this property.